guides/common/modules/ref_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Managing_Packages_on_the_Base_Operating_System_{context}"]
+= Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}
+
+{Project} provides the `{foreman-maintain} packages` command to install and update packages on the base operating system of your {ProjectServer} or {SmartProxyServers}.
+
+[WARNING]
+====
+Installing and updating individual packages with `yum` is not supported.
+While {Project} provides the `{foreman-maintain} packages unlock` command to bypass `{foreman-maintain} packages` and enable package management with `yum`, this can lead to package inconsistencies and compromise the stability of the system.
+Always use `{foreman-maintain} packages` to manage packages.
+====
+
+After you install or update a package on the base operating system, running `{foreman-installer}` afterwards is required.
+When you manage packages with `{foreman-maintain} packages`, the command automatically runs `{foreman-installer}` as well.
+Note that this results in restarting some services on the base operating system.
+
+.Overview of `{foreman-maintain} packages` subcommands
+[cols=2,1,options="header"]
+|====
+|Subcommand |Description
+a|
+[options="nowrap", subs="+quotes,attributes"]
+----
+{project-package-install} _package_1_ _package_2_
```

Comment:
```suggestion
{project-package-install} _My_Package_1_ _My_Package_2_
```

Reply:
Applied. I'm somewhat concerned about what it looks like in the preview. I'm thinking that I could perhaps drop `My_Package_2` to make the code blocks a bit shorter. @maximiliankolb

Reply:
Yeah, you could either use `_My_Package_` or `_My_Packages_` as a shorter placeholder.

Reply:
Thanks, let's go with `My_Package`.